common/djangoapps/third_party_auth/samlproviderdata/tests/test_samlproviderdata.py

```diff
@@ -141,16 +142,42 @@ def test_patch_one_provider_data(self):
 
     def test_delete_one_provider_data(self):
         # DELETE auth/saml/v0/providerdata/ -d data
-        url = reverse('saml_provider_data-detail', kwargs={'pk': self.saml_provider_data.id})
-        data = {}
-        data['enterprise_customer_uuid'] = ENTERPRISE_ID
+        url_base = reverse('saml_provider_data-detail', kwargs={'pk': self.saml_provider_data.id})
+        query_kwargs = {'enterprise_customer_uuid': ENTERPRISE_ID}
+        url = '{}?{}'.format(url_base, urlencode(query_kwargs))
```

Comment:
Is this the preferred method of making a delete request with data? Or did something change with the endpoint to require this?

Reply:
Django Rest Framework does not support a body in Delete requests. The format of this test is to append the uuid into the body of the request, which we actually cannot do. How this works in tests, but not reality, no idea.

Reply:
yeah we will go with the url param method for now, so DELETE auth/saml/v0/provider_config/:id/?enterprise_customer_uuid=id

i agree it is odd to have query string in a delete, not disallowed by http but yes odd. It is due to rbac requirement for now and that we are not able to send in the that extra piece of info right now without doing this, we can revisit in future